pages/glossary/section-of-content.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Section of content
 key: section-of-content
+unambiguous: true
```

Comment:
I think this is ambiguous. "A distinct part or subdivision of a document" is pretty vague and can mean a lot of different things. For example, visually distinct, an element could be a "distinct part", things in a landmark, etc.

Reply:
Hmm… You might be right 😢 

The definition is currently used in the expectation of https://act-rules.github.io/rules/b49b2e so this need to be fixed somehow (and I'm making a heavy use of it in the Bypass block PR also…)

The fact that a `p` element might, or maybe not, be a section of content in itself doesn't really bother me. I see that as subjectivity, not ambiguity (i.e., yes, a `p` can be a section of content and its up to each of us to decide whether this specific `p` in this specific page is).